src/components/link/link-story-angular.ts

```diff
@@ -13,7 +13,7 @@ import baseStory, { Default as baseDefault } from './link-story';
 
 export const Default = args => ({
   template: `
-    <bx-link [disabled]="disabled" href="href" (click)="onClick($event)">
+    <bx-link [disabled]="disabled" href={{href}} (click)="onClick($event)" size={{size}}>
```

Comment:
We standardize on property assignments for Angular:

```suggestion
    <bx-link [disabled]="disabled" [href]="href" [size]="size" (click)="onClick($event)">
```

Reply:
Got it, thanks @asudoh!